huaweicloud/config/config.go

```diff
@@ -515,6 +515,14 @@ func (c *Config) EnterpriseProjectClient(region string) (*golangsdk.ServiceClien
 	return c.NewServiceClient("eps", region)
 }
 
+func (c *Config) TmsV1Client() (*golangsdk.ServiceClient, error) {
```

Comment:
`region` should be specified although it is global.

Reply:
done.